test/acceptance/features/steps/steps.py

```diff
@@ -190,6 +206,78 @@ def then_sbo_jq_is(context, jq_expression, sbr_name, json_value_regex):
+    """
+    serverless_operator = ServerlessOperator()
+    if not serverless_operator.is_running():
+        print("Serverless operator is not installed, installing...")
+        serverless_operator.install_operator_subscription() | should.be_truthy.desc(
+            "serverless operator subscription installed")
+        serverless_operator.is_running(wait=True) | should.be_truthy.desc("serverless operator installed")
+    print("Serverless operator is running!!!")
+    context.serverless_operator = serverless_operator
+
+
+# STEP
+is_quarkus_s2i_image_present_step = u'Quarkus s2i builder image is present'
+
+
+@given(is_quarkus_s2i_image_present_step)
+def given_quarkus_builder_image_is_present(context):
+    """
+    Checks if quarkus s2i builder image is present
+    """
+    builder_image = BuilderImage()
+    if not builder_image.is_present():
+        builder_image.import_and_patch() | should.be_truthy.desc("Quarkus image is imported from image registry and patched to builder")
+        builder_image.is_present() | should.be_truthy.desc("Quarkus image is present")
+    print("Quarkus s2i builder image is present!!!")
+
+
+# STEP
+is_knative_serving_running_step = u'Knative serving is running'
+
+
+@given(is_knative_serving_running_step)
+def given_knative_serving_is_running(context):
+    """
+    creates a KnativeServing object to install Knative Serving using the OpenShift Serverless Operator.
+    """
+    knative_namespace = Namespace("knative-serving")
+    knative_namespace.create() | should.be_truthy.desc("knative serving namespace is created")
+    Namespace(context.namespace.name).switch_to() | should.be_truthy.desc("Switch back to the context namespace")
+    knative_serving = KnativeServing(namespace=knative_namespace.name)
+    if not knative_serving.is_present():
+        print("knative serving is not present, create knative serving")
+        knative_serving.create() | should.be_truthy.desc("knative serving is created")
+        knative_serving.is_present() | should.be_truthy.desc("knative serving is present")
+    print("knative serving is present!!!")
+
+
+# STEP
+quarkus_app_is_imported_step = u'Quarkus application "{application_name}" is imported as Knative service'
+
+
+@given(quarkus_app_is_imported_step)
+@when(quarkus_app_is_imported_step)
+def quarkus_app_is_imported_as_knative_service(context, application_name):
+    namespace = context.namespace
+    application = QuarkusApplication(application_name, namespace.name)
+    if not application.is_imported():
+        print("application is not imported, trying to import it")
+        application.install() | should.be_truthy.desc("Quarkus application is installed")
+        application.is_imported() | should.be_truthy.desc("Quarkus application is imported")
+    context.application_original_generation = application.get_observed_generation()
+    context.application = application
+    context.application_type = "knative"
+
```

Comment:
see above comments for this, they apply here as well.

Reply:
I have replaced should with assert with corresponding error message.